cactusbot/commands/magic/social.py

```diff
@@ -13,32 +11,55 @@ class Social(Command):
     COMMAND = "social"
 
     @Command.command(hidden=True)
-    async def default(self, *args: False, channel: "channel"):
+    async def default(self, *services: False):
         """Get a social service if it's provived, or give it all."""
 
-        social_data = await (await aiohttp.get(
-            "https://beam.pro/api/v1/channels/{}".format(channel)
-        )).json()
+        response = []
+        if services:
+            for service in services:
+                social = await self.api.get_social(service)
+                if social.status == 200:
+                    data = await social.json()
+                    response.append(
+                        data["data"]["attributes"]["service"].title() + ': ')
+                    response.append(
+                        ("url", data["data"]["attributes"]["url"]))
+                    response.append(', ')
+                else:
+                    return "'{}' not found on the streamer's profile!".format(
+                        ', '.join(services))
```

Comment:
Shouldn't `.join(services)` be `.join(service)`? Some of the services might not exist, just not this current one